snuba/web/split.py

```diff
@@ -128,9 +173,51 @@ def time_split(dataset, request: Request, *args, **kwargs):
 
         return overall_result
 
-    def col_split(
-        dataset, request: Request, column_split_spec: ColumnSplitSpec, *args, **kwargs
-    ):
+
+class ColumnSplitSpec(NamedTuple):
+    """
+    Provides the column names needed to perform column splitting.
```

Comment:
It's also used for the time splitting not just column? (in the case of timestamp_column)